blobfuse/directoryapis.cpp

```diff
@@ -142,6 +159,46 @@ int azs_readdir(const char *path, void *buf, fuse_fill_dir_t filler, off_t, stru
                         stbuf.st_nlink = 1;
                         stbuf.st_size = listResults[i].content_length;
                         fillerResult = filler(buf, prev_token_str.c_str(), &stbuf, 0); // TODO: Add stat information.  Consider FUSE_FILL_DIR_PLUS.
+
+                        /// List attribute cache: cache the found file locally without the contents
```

Comment:
you are not updating attributes of already cached files, is it expected behavior?

Reply:
You are right. This solution is missing changes on file size that may happen from other nodes... No data corruption, but until you open the file, you keep seeing the outdated file size. I will address this.

Reply:
As discussed offline - we decided that upcoming garbage collection feature will address this. There will be no cached files on disk that go beyond the file_cache_timeout_in_seconds